colour/appearance/ciecam02.py

```diff
@@ -1344,7 +1346,8 @@ def P(N_c, N_cb, e_t, t, A, N_bb):
     A = np.asarray(A)
     N_bb = np.asarray(N_bb)
 
-    P_1 = ((50000 / 13) * N_c * N_cb * e_t) / t
+    with np.errstate(divide='ignore'):
```

Comment:
I think warnings should only be muted if their effects are explicitly handled and invisible to the caller.